kernel/src/generic_main.rs

```diff
@@ -26,10 +28,33 @@ pub fn generic_main<const LAUNCH_TESTS: bool>(dt: DeviceTree, hacky_devices: &[&
     // Driver stuff
     // let _drvmgr = DriverManager::with_devices(&dt).unwrap();
 
-    hal::irq::init_irq_chip((), &globals::PHYSICAL_MEMORY_MANAGER)
+    log::trace!("mapping gic pages");
+
+    cfg_if::cfg_if! {
+        if #[cfg(target_arch = "aarch64")] {
+            let (gicd_base, gicc_base) = (0x800_0000, 0x801_0000);
+            HAL.kpt().lock().identity_map_range(
+                VAddr::new(gicd_base),
+                0x0001_0000 / HAL.page_size(),
```

Comment:
I would rather have a calculus or a const than just `0x0001_0000`

Reply:
A calculus ?

Reply:
Looks like this comment was on an older version of this branch which was mistakenly pushed, I pushed the actual revision.

Reply:
(te current code does this by looking up the gic node in the device tree)